frontend/components/commons/SideBar.vue

```diff
@@ -192,6 +73,9 @@ export default {
       this.openExportModal = false;
       this.exportAnnotations({ name: this.dataset.name });
     },
+    toggleVisibleMetrics() {
+      !this.visible ? (this.visible = true) : (this.visible = false);
```

Comment:
if `this.visible` if properly initialized, could we set as follows:
```suggestion
      this.visible  = !this.visible;
```